src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlHelperPartialExtensions.cs

```diff
@@ -241,6 +241,94 @@ public static class HtmlHelperPartialExtensions
         /// <param name="partialViewName">
         /// The name of the partial view used to create the HTML markup. Must not be <c>null</c>.
```

Comment:
Feels weird to call out the non-nullability of parameters. We don't do that elsewhere.